csunplugged/templates/dev/index.html

```diff
@@ -80,6 +81,19 @@
     </ul>
   {% endif %}
 
+  {% if glossary_terms %}
+    <h1>Glossary Terms</h1>
+    <ul>
+      {% for glossary_term in glossary_terms %}
+        <li>
+          <h3>{{ glossary_term.term }}</h3>
+          <p>Key: <code>{{ glossary_term.slug }}</code></p>
+          {% render_html_field glossary_term.definition %}
+        </li>
+      {% endfor %}
+    </ul>
+  {% endif %}
```

Comment:
I am slightly concerned that this is going to cause the dev page to get way too long, can the layout of this please be compacted down somehow (not a big title for each word maybe? just bolded text?) and put at the bottom of the page.